tests/test_fin_builder.py

```diff
@@ -54,46 +54,39 @@ def add_fin_find_compile_job(session_id):
   counts = import_cfgs(args, dbt)
 
   #load jobs
+  print('Loading jobs')
   args = LdJobArgs
   args.label = 'tuna_pytest_fin_builder'
   args.tag = 'test_fin_builder'
   args.fin_steps = ['miopen_find_compile', 'miopen_find_eval']
   args.session_id = session_id
 
   connect_db()
-  counts = {}
-  counts['cnt_jobs'] = 0
-  dbt = DBTables(session_id=None, config_type=args.config_type)
-  if args.tag:
-    try:
-      tag_name_test(args.tag, dbt)
-    except ValueError as terr:
-      print(terr)
+  #if args.tag:
+  #  try:
+  #    tag_name_test(args.tag, dbt)
+  #  except ValueError as terr:
+  #    print(terr)
 
-  add_jobs(args, counts, dbt)
+  return add_jobs(args, dbt), dbt
 
 
 def test_fin_builder():
   args = GoFishArgs()
   machine_lst = load_machines(args)
   machine = machine_lst[0]
 
-  args.session_id = add_test_session()
+  #args.session_id = add_test_session()
+  args.session_id = 1
```

Comment:
the problem is that test_fin_eval also needs the same session. And since this session_id here is dynamic, test_fin_eval would not know what it is. So hardcoding it enables using the same session_id in both test. Alternatively i could also get the session_id from the table in the eval test and use it that way. 
Which way do you prefer?